### DISCUSSION TITLE
Connection hangs after some time (FPS 0)

### DISCUSSION THREAD

**Discussion by 64kramsystem** [rustdesk/rustdesk#6800] [General]
### Bug Description

After some time (say, an hour), the connection hangs, without any apparent reason.

The quality monitor still updates the information, but I can see `FPS 0`.

I think that input is actually still received; if this is correct, then it's just that the screen image sent to the client is not updated anymore.

If I disconnect and reconnect, the connection works.

I use RD regularly, and this is a recurring bug.

### How to Reproduce

Stay connected for a long time. At some point, updates won't be visible anymore, and the screen displayed will hang.

### Expected Behavior

The connection shouldn't hang.

### Operating system(s) on local side and remote side

Linux -> Linux

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

-

**Comment by rustdesk** [comment]
Please self-hosting

**Reply by 64kramsystem** [reply]
I've converted to a self-hosted solution, and the problem still happens, although much less.

The server is on a local network, and the connection is stable, so it's not a network connection problem; additionally, if it were, input should stop working, but instead it does - only the image hangs.

**Comment by 64kramsystem** [comment]
I haven't been experiencing this for a long time (self-hosting), so all good for me.
